frontend/pages/project-developers/new.tsx

```diff
@@ -287,39 +274,24 @@ const ProjectDeveloper: PageComponent<ProjectDeveloperProps, NakedPageLayoutProp
               <p className="font-sans font-medium text-base text-gray-600 mb-4.5">
                 <FormattedMessage defaultMessage="General" id="1iEPTM" />
               </p>
-              <div>
-                <label
-                  htmlFor="picture"
-                  className="inline-block mb-4 mr-2.5 text-sm font-semibold text-gray-800 text-primary"
-                >
+              <div className="mb-3">
+                <Label htmlFor="picture" className="mr-2.5">
                   <FormattedMessage defaultMessage="Picture" id="wvoA3H" />
-                </label>
+                </Label>
                 <FieldInfo
                   infoText={formatMessage({
                     defaultMessage: 'Add your logo or a picture that indentifies the account.',
                     id: 'bjubNC',
                   })}
                 />
               </div>
-              <div className="flex items-center justify-start">
-                <Image
-                  src={imagePreview || '/images/avatar.svg'}
-                  width={48}
-                  height={48}
-                  className="rounded-full"
-                  alt={formatMessage({ defaultMessage: 'profile image', id: 'OX08fr' })}
-                />
-                <input
-                  id="picture"
-                  className="ml-5"
-                  name="picture"
-                  type="file"
-                  accept="image/png, image/jpeg"
-                  {...register('picture')}
-                  onChange={handleUploadImage}
-                  aria-describedby="picture-error"
-                />
-              </div>
+              <ImageUploader
+                name="picture"
+                id="picture"
+                register={register}
+                preview
+                handleChangeImage={() => clearErrors(['picture'])}
```

Comment:
🟠

If the component is correctly registered with React Hook Form, this shouldn't be needed, so you could probably remove the `handleChangeImage` prop from the `<ImageUploader />` component.